### ISSUE TITLE
'standardized' in definition of listNym

### ISSUE THREAD

**Issue by stuartyeates** [TEIC/TEI#2150]
Since at least P5 1.0.0 the definition of listNym has been:

    <listNym> (list of canonical names) contains a list of nyms, that is, standardized names for any thing.

But nowhere in the discussion of nym or listNym is there any mention of standards or standardization, only canonicalness.  Additionally this gives the (I believe mistaken) suggestion that all of the nyms in a nymList must be names for the same thing. The definition should be:

    <listNym> (list of canonical names) contains a list of nyms, that is, canonical names for things.

**Comment by sydb** [comment]
I think you have a point.

**Comment by lb42** [comment]
"normalized" would work too, maybe better, since "canonised/canonical" has undesirable religious/litcrit overtones. And maybe adding a `remarks` clarifying that no particular normalization method is proposed by the Guidelines, though a project would be well advised to make their principles explicit.

**Comment by stuartyeates** [comment]
Switching from "canonised/canonical" to "normalized" would probably require switching the discussion of the tag in the chapter too.

"Standard/ized" is used a _lot_ in the chapter to refer to date standardization.

**Comment by lb42** [comment]
Yes. But there ARE standards for date representation ... Several in fact.

**Comment by bansp** [comment]
I am not sure if Lou's perceived overtones for "canonical" are shared by everyone -- I think, as a speaker of "near native technical English", I might have an issue with "canonised", but not "canonical", which -- for many, I think -- functions across the board as referring to established/institutionalized practice, without digging deeper into the semantics/etymology of "canon".

Using "canonical" for both the gloss and the description feels like a welcome tautology -- because the gloss here is supposed to just make the description more concise, rather than to serve as a definiendum. Using "standardized" in the description feels like defining "canonical", and defining it rather badly, for at least those of us involved into standardization processes.

I would go for Stuart's suggestion, because then it is just a correction of a slip in the description.

**Comment by lb42** [comment]
My perceptions are rarely shared by everyone, though I hope often by many! But I agree that the gloss and the description should be consistent, whatever word is used and that "standardized" is freight with unwarranted expectations. In my view there is no need for a gloss, since "listNym" just means "a list of nyms" whatever "nyms" may be: it is not the place where "nyms" should be defined or explained. The gloss element throughout the Guidelines is not meant to do more than expand on those few elements whose names are not self-explanatory.

**Comment by bansp** [comment]
> My perceptions are rarely shared by everyone, though I hope often by many!

I share that hope, and value your perceptions. :-)

> The gloss element throughout the Guidelines is not meant to do more than expand on those few elements whose names are not self-explanatory.

And thanks for that, I recall having aesthetical issues regarding the (I thought) obligatory use of glosses, when editing my ODDs or ODD proposals. I'll bear that in mind.

**Comment by PFSchaffner** [comment]
Ah the joys of nuance. FWIW, I would place these words roughly in this order of generality: regularized, normalized, standardized, canonical. That is, 'regularized' is the loosest and simply means 'consistent, in accordance with some rule,' however arbitrary; 'normalized' goes a step further and implies that that rule carries the weight of custom or authority; 'standardized' drops the moral authority of 'normalized' but implies consistency with some formal standard, though that standard may be local and capricious; and 'canonical' is the strictest of all, since it implies conformity with a universal canon, not just any rule but a rule having legal and quasi-sacred authority--at the least, "of admitted authority, excellence, or supremacy; authoritative; orthodox" (OED).

**Comment by bansp** [comment]
It seems then that I should ignore my non-native understanding of "canonical" and vote for "normalized" throughout (if the gloss stays).

**Comment by PFSchaffner** [comment]
I suspect also that there is probably some conflict at work between 'humanist' English and 'techie' English.

**Comment by stuartyeates** [comment]
Unicode, which both XML and TEI P5 are built on, uses 'canonical'  and 'normalized' in specific ways. Depending on how aligned we want to be with them, we may wish to be influenced by this. See https://unicode.org/reports/tr15/
